data/countries_grammar.csv

```diff
@@ -189,7 +189,7 @@ SYR,,1,,,,,,,,,
 TCA,1,1,,,,,,,,,
 TCD,,,,,,,,,,,
 TGO,,,,,,,,,,,
-TKL,,,,,,,,,,,
+TKL,,,1,,,,,,,
```

Comment:
@abigailmarshall here it looks like, the comma was simply replaced by the '1', leaving it with 11 columns only where it should have 12 columns (11 commas total)